integration-tests/sanity-functional-protractor/page_objects/api-po.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const AbstractPage = require('./abstract-po.js');
+const qs = require('querystring')
+const chakram = require('chakram');
+const port = 8181;
+const getMetadata = '/api/metadata';
+const getArtifact = '/api/artifact';
+
+function RestAPI() {
```

Comment:
Why are we mixing two different frameworks here? REST or API in general has nothing to do with page objects. These different layers should be tested in separation.
Does this open a browser? If so, please plug it out, it's an unnecessary overhead for the purpose of this test.

Reply:
true! idea was to keep all in one module but as your said is overhead to open browser for API requests. So I've excluded api from protractor tests and moved to separate module: 'api-tests'.
And now I'm using chakram with pure jamsmine.